FAIMS/FAIMS3
# feat: email is used as username in forms

# feat: email is used as username in forms

## JIRA Ticket

[BSS-407](https://jira.csiro.au/browse/BSS-407)

## Description

There is no need to expose a distinct username field for users in the application. 

This PR removes the username input in all UIs, and changes them to email. The email is used as the username in existing workflows. This allows the admin as username workflow to continue functioning. 

I also updated touched methods with Zod validators in the `@faims3/data-model` package. 

## How to Test

Deploy a local instance, check that registration and login forms function and show email appropriately. Ensure that both email login and admin style username login functions. Ensure API unit tests run (CI/CD should show this).

@stevecassidy  if you have a working google auth setup, it would be good if you can check that is not effected. 

## Checklist
